GraphQL_Express_Server/graphql/resolvers.js

```diff
@@ -1,5 +1,20 @@
+const { SensorFault } = require("../models/sensorfault");
+
 module.exports = {
     Query: {
         MeanClimateMeasurements: (_, {start, stop}, { dataSources }) => dataSources.influx.getMeanClimateValues(start, stop),
+        SensorFaults: () => SensorFault.find({
```

Comment:
Do you think we could add a datasaource object and call a method from that object when the resolver is invoked?